lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/rcgnmi/GnmiConnectionITTest.java

```diff
@@ -203,6 +209,87 @@ public void connectDeviceCorrectlyTest()
         assertTrue(disconnectDevice(GNMI_NODE_ID));
     }
 
+    @Test
+    public void connectDeviceWithAdditionalCapabilityAndModelTest()
+        throws InterruptedException, IOException, ExecutionException, TimeoutException {
+        final HttpResponse<String> getGnmiTopologyResponse = sendGetRequestJSON(GNMI_TOPOLOGY_PATH);
+        assertEquals(HttpURLConnection.HTTP_OK, getGnmiTopologyResponse.statusCode());
+        final JSONArray topologies =
+            new JSONObject(getGnmiTopologyResponse.body()).getJSONArray("network-topology:topology");
+        assertEquals(1, topologies.length());
+        final JSONObject gnmiTopologyJSON = topologies.getJSONObject(0);
+        LOG.info("Empty gnmi-topology check response: {}", gnmiTopologyJSON);
```

Comment:
remove the log or rather use the message parameter in `assertThrows` (as this log relates to that assertion that there are no nodes)